acl/tests/test.bats

```diff
@@ -0,0 +1,31 @@
+failWithMessage() {
+    local rest="$@"
+    diff <(echo "actual result is below") <(echo "${rest}")
+}
+
+# parse the package identifier
+IFS=/ read -r -a _pkg_ident_array <<< "${TEST_PKG_IDENT}"
```

Comment:
Instead of doing lines 7-11 here, you should be able to simply do:

```
TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
```

Reply:
Amended.